supervision/detection/core.py

```diff
@@ -187,10 +187,18 @@ def from_yolov8(cls, yolov8_results) -> Detections:
             >>> detections = sv.Detections.from_yolov8(result)
             ```
         """
+        masks = None
+        if yolov8_results.masks:
+            from ultralytics.yolo.utils.ops import scale_image
```

Comment:
Hi 👋🏻 @hardikdava! We can't use `ultralytics` code directly like that :/ The moment we will do it, we need to change the LICENSE of `supervision` to a very restrictive YOLOv8 LICENSE.

But I took a look at the `scale_image` implementation. A lot of it is padding logic that we don't use. And if I'm not mistaken, all we need is :

```python
masks = cv2.resize(masks, (im0_shape[1], im0_shape[0]))
```
Am I right?

Reply:
Yes, @SkalskiP . I tested by just using `cv2.resize` function. It works fine without using any functionality from `ultralytics`. I will make necessary changes.